internal/zinc-scripted/src/main/contraband-scala/sbt/internal/inctest/Project.scala

```diff
@@ -17,7 +17,7 @@ final class Project private (
     case _ => false
   }
   override def hashCode: Int = {
-    37 * (37 * (37 * (37 * (17 + name.##) + dependsOn.##) + in.##) + scalaVersion.##)
+    37 * (37 * (37 * (37 * (37 * (17 + "Project".##) + name.##) + dependsOn.##) + in.##) + scalaVersion.##)
```

Comment:
Is this the new generated code? If so, this is still incorrect, you should use the hash of the fully qualified name of `Project`. Otherwise, you can still have predictable collisions of the hash with other classes named `Project`.